server/api/routes/user.js

```diff
@@ -16,7 +16,7 @@ router.get(
   userController.getUserListingViaUserAndListingID
 );
 router.put(
-  '/:id/listing/:lid',
+  '/:uid/listing/:id',
```

Comment:
hmm I don't think we need to change this, as `id` in `user.js` would be assumed to be user id instead of listing id. lol